client/components/web-preview/content.jsx

```diff
@@ -0,0 +1,308 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import debugModule from 'debug';
+import noop from 'lodash/noop';
+import page from 'page';
+import shallowCompare from 'react-addons-shallow-compare';
+import { v4 as uuid } from 'uuid';
+import addQueryArgs from 'lib/route/add-query-args';
+
+/**
+ * Internal dependencies
+ */
+import Toolbar from './toolbar';
+import touchDetect from 'lib/touch-detect';
+import { isMobile } from 'lib/viewport';
+import { localize } from 'i18n-calypso';
+import Spinner from 'components/spinner';
+import SeoPreviewPane from 'components/seo-preview-pane';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+const debug = debugModule( 'calypso:web-preview' );
+
+export class WebPreviewContent extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.previewId = uuid();
+
+		this._hasTouch = false;
+		this._isMobile = false;
+
+		this.state = {
+			iframeUrl: null,
+			device: props.defaultViewportDevice || 'computer',
+			loaded: false
+		};
+
+		this.setIframeInstance = ref => this.iframe = ref;
+
+		this.selectSEO = this.setDeviceViewport.bind( this, 'seo' );
+		this.setDeviceViewport = this.setDeviceViewport.bind( this );
```

Comment:
For most of these, you can just use fat arrow class methods. So, instead of explicitly binding in the constructor, just declare like:
```
setDeviceViewport = ( device = 'computer' ) => {
// ...
}
```

,,,& `this` will be bound to the component